### DISCUSSION TITLE
One-Click app for BTCPAY

### DISCUSSION THREAD

**Discussion by antonind72** [coollabsio/coolify#4292] [Service Template Requests]
BTC PAY is a self-hosted cryptocurrency payment solution and it would be great to have it as a one-click installation app.

link : https://btcpayserver.org/
github : https://github.com/btcpayserver

**Comment by mredodos** [comment]
+1

https://github.com/btcpayserver/btcpayserver-docker

**Comment by felix-schindler** [comment]
#1473
